tests/virtualization/universal/hotplugging.pm

```diff
@@ -203,11 +203,12 @@ sub set_guest_memory {
 
 sub test_vmem_change {
     my $guest = shift;
-    my ($sles_running_version, $sles_running_sp) = get_os_release;
-
-    if (get_var('VIRT_AUTOTEST') && ($sles_running_version lt '12' or ($sles_running_version eq '12' and $sles_running_sp lt '3'))) {
-        record_info('Skip memory hotplugging on outdated before-12-SP3 SLES product because immature memory handling situations');
-        return;
+    if (is_sle) {
```

Comment:
What is wrong on openSUSE here?

Reply:
openSUSE TW need not skip memory hotplugging as it has most recent virtualization packages. Meanwhile, get_os_release is not apply to opensuse because of the different version format.

# cat /etc/issue
Welcome to openSUSE Tumbleweed 20210521 - Kernel \r (\l).